submissions/models.py

```diff
@@ -99,7 +99,12 @@ class Submission(models.Model):
     created_at = models.DateTimeField(editable=False, default=now, db_index=True)
 
     # The answer (JSON-serialized)
-    raw_answer = models.TextField(blank=True)
+    # NOTE: previously, this field was a TextField named `raw_answer`.
```

Comment:
Just to be certain, you are saying that raw_answer was a text field that stored JSON values, and thus there are no migration concerns to switching it to a JSONField.